bumble/host.py

```diff
@@ -21,7 +21,8 @@
 import logging
 import struct
 
-from typing import Optional
+from collections.abc import Mapping, Callable, Awaitable
```

Comment:
`Callable` and `Awaitable` now exist in the `typing` module. `Mapping` exists there too, but maybe we can use the more direct `Dict` type here? (I think the rule of thumb for `Dict` vs `Mapping` is to use `Mapping` for function arguments if possibe (to make it more flexible for the caller), but use `Dict` for return parameters or class members, to make it more precise.

Reply:
Done